### DISCUSSION TITLE
稍后再看和播放一个视频列表时候倍速扩展不生效

### DISCUSSION THREAD

**Discussion by TonyShitDark** [the1812/Bilibili-Evolved#3998] [提问 / Q&A]
如题 稍后再看和播放一个视频列表时候，倍速扩展不生效
以及下载视频的功能也不显示，只有单独播放一个视频时候这些功能才生效
之前的版本中倍速扩展是可以在这些地方生效的

**Comment by Geeyun-JY3** [comment]
请善用搜索，看置顶：#3916
